modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -13,25 +46,28 @@ You can configure an {product-title} cluster
 ifeval::["{context}" == "load-balancing-openstack"]
 on {rh-openstack-first}
 endif::[]
-to use an external load balancer in place of the default load balancer.
-
-// Maybe an About mod in support
 
+You can also configure an {product-title} cluster to use an external load balancer that supports multiple subnets. If you use multiple subnets, you can explicitly list all the IP addresses in any networks that are used by your load balancer targets. This configuration can reduce maintenance overhead because you can create and destroy nodes within those networks without reconfiguring the load balancer targets.
 
+If you deploy your ingress pods by using a machine set on a smaller network, such as a `/27` or `/28`, you can simplify your load balancer targets.
 
 .Prerequisites
 
-* On your load balancer, TCP over ports 6443, 443, and 80 must be available to any users of your system.
+* On your load balancer, TCP over ports 22623, 6443, 443, and 80 must be reachable by all users of the system that are located outside the cluster.
```

Comment:
can we use the original wording of "users of your system"?

Reply:
Thanks. Yes, that is more specific. Updated.